### PR TITLE
 Gap on the page from 913px to 971px on page: layer5.io/docker-extension-meshery is solved

### LINKED ISSUE #3361: A gap on the page from 913px to 971px on page: layer5.io/docker-extension-meshery
#### Description

A gap on the page from 913px to 971px on page: layer5.io/docker-extension-meshery
Screenshot 2022-10-18 013010

#### Environment:
- Host OS:  Windows
- Browser: Chrome

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
 Gap on the page from 913px to 971px on page: layer5.io/docker-extension-meshery is solved
This PR fixes #3361 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by UtkarshMishra12** [comment]
@Nikhil-Ladha I have made the changes, let me know your thoughts about it. 😊

**Comment by l5io** [comment]
🚀 Preview for commit feeb9995cd032b7a1d972899cbcf67bc4b247cd2 at: https://635046fb4a2c3f00b4fbdb46--layer5.netlify.app

File: src/sections/General/Navigation/navigation.style.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ const NavigationWrap = styled.header`
 
   .nav-container {
     display: flex;
+    @media (min-width:768px){
```
Comment (Nikhil-Ladha) [inline src/sections/General/Navigation/navigation.style.js:14]: It should be limited from `912px to 992px` as for other screen resolutions it is constrained by the `Container` component.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by UtkarshMishra12** [comment]
Do I have to replace the above line with this line? @Nikhil-Ladha 
@ media (min-width:912px) and (max-width:992px){
    max-width:850px;
}

**Comment by Nikhil-Ladha** [comment]
> Do I have to replace the above line with this line? @Nikhil-Ladha @ media (min-width:912px) and (max-width:992px){ max-width:850px; }

Yup

**Comment by l5io** [comment]
🚀 Preview for commit 4cc919b9ed42d05a6dacfdf99591f4c972675c61 at: https://6350d9448ccbf0695e7ecc77--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**